src/service/viewport-impl.js

```diff
@@ -1041,12 +1041,29 @@ export class ViewportBindingNatural_ {
   disableScroll() {
     this.win.document.documentElement.classList.add(
         'i-amphtml-scroll-disabled');
+    // Unfortunately overflow hidden is not working in ios natural viewport.
+    if (platformFor(this.win).isIos()) {
+      this.win.document.addEventListener('touchmove', this.handleTouchMove_);
+    }
   }
 
   /** @override */
   resetScroll() {
     this.win.document.documentElement.classList.remove(
         'i-amphtml-scroll-disabled');
+    // Unfortunately overflow hidden is not working in ios natural viewport.
```

Comment:
Do we still need ` this.win.document.documentElement.classList.remove(
          'i-amphtml-scroll-disabled');		          'i-amphtml-scroll-disabled');`?